js/form.js

```diff
@@ -0,0 +1,23 @@
+const form = document.getElementById('projectForm');
+
+function lowercaseEmail(input) {
+  return input === String(input).toLowerCase();
+}
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
```

Comment:
- [ ] Good job so far 👏  Unfortunately, as the previous reviewer said, the form is not submitted when the validations pass. I think it is because you are preventing it from submitting at the beginning.

HINT 💡: As a hint, I suggest you put this line of code (ie: event.preventDefault()) in the condition block (maybe if statement) when the validations are not passing. 😉 😎 👍  To be clear, the form will be prevented from submitting (e.preventDefault()) only when the validations are not passing otherwise it should submit as normal 🏃